src/components/Tweet.css

```diff
@@ -10,3 +10,16 @@
 .Tweet__header {
   position: relative;
 }
+
+.Tweet_image {
+  border-radius: 5px;
+  /*width: 95%;*/
+  float: right;
```

Comment:
To make it look nicer I would stick with a fixed width for the image to ensure it fits in view. If you set a width on an image, the browser will force it to be those dimensions. Only specifying width will mean the height adjusts in proportion.